APMrover2/GCS_Mavlink.cpp

```diff
@@ -289,8 +289,8 @@ void Rover::send_wind(mavlink_channel_t chan)
     // send wind
     mavlink_msg_wind_send(
         chan,
-        degrees(rover.g2.windvane.get_apparent_wind_direction_rad()),
-        0,
+        wrap_360(degrees(g2.windvane.get_absolute_wind_direction_rad())),
```

Comment:
I think Plane also reports an absolute wind direction estimate (i.e. "wind is from the North" rather than "Wind is from straight ahead") so  the only divergence here is that Plane is reporting West as -90 degrees while Rover/Boat would report it as 270degrees.  I think reporting it as 270 is better actually... I understand that diverging is not good but.. I don't think reporting absolute directions using negatives is very intuitive.

This is the header for the EKF2 wind estimate function used by plane
> // return the NED wind speed estimates in m/s (positive is air moving in the direction of the axis)
> // An out of range instance (eg -1) returns data for the the primary instance
> void getWind(int8_t instance, Vector3f &wind) const;